sigar_windows.go

```diff
@@ -141,7 +141,42 @@ func (self *FileSystemList) Get() error {
 }
 
 func (self *DiskList) Get() error {
-	return notImplemented()
+	diskQueries := []string{
+		`\physicaldisk(*)\disk reads/sec`,
+		`\physicaldisk(*)\disk read bytes/sec`,
+		`\physicaldisk(*)\% disk read time`,
+		`\physicaldisk(*)\disk writes/sec`,
+		`\physicaldisk(*)\disk write bytes/sec`,
+		`\physicaldisk(*)\% disk write time`,
+	}
+
+	// Run a PDH query for metrics across all physical disks
+	queryResults, err := runRawPdhArrayQueries(diskQueries)
+	if err != nil {
+		return err
+	}
+
+	self.List = make(map[string]DiskIo)
+	for disk, counters := range queryResults {
+		if disk == "_Total" {
+			continue
+		}
+
+		self.List[disk] = DiskIo {
+			ReadOps: uint64(counters[0]),
+			ReadBytes: uint64(counters[1]),
+			// `% disk read time` is measured in 100ns ticks, 
+			// divide by 10000 to get milliseconds from the raw counter
+			ReadTimeMs: uint64(counters[2]/10000),
```

Comment:
I'm a little confused on the math here.  Does `ReadTimeMS` mean an absolute number of milliseconds spent performing read operations or a percentage?